lib/bolt/puppetfile.rb

```diff
@@ -56,9 +56,10 @@ def self.parse(path, skip_unsupported_modules: false)
     # Writes a Puppetfile that includes specifications for each of the
     # modules.
     #
-    def write(path)
+    def write(path, moduledir = nil)
       File.open(path, 'w') do |file|
         file.puts '# This Puppetfile is managed by Bolt. Do not edit.'
+        file.puts "moduledir '#{moduledir}'" if moduledir
```

Comment:
```suggestion
        file.puts "moduledir '#{File.basename(moduledir)}'" if moduledir
```

We should put the basename here, not the full path, otherwise the project can't be shared between users.